shared.chains.json

```diff
@@ -5,253 +5,49 @@
         "name": "ENSRegistry",
         "address": "0xe7410170f87102DF0055eB195163A03B7F2Bff4A"
       },
-      "0x573ce922ef30ea27239add0fb9f2af8d008314dc4a0b3eaefa0adaf8963e44fa": {
```

Comment:
Deleted everything to have a clean slate and anyway, most of the old stuff was never going to be reused